tests/ui/test_search.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from pages.desktop.home import Home
+
+
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_that_searching_for_addon_returns_addon_as_first_result(
+        my_base_url, selenium, addon):
+    """Test searching for an addon returns the addon."""
+    home_page = Home(selenium, my_base_url).open()
+    name = home_page.featured_extensions.extensions[0].name
+    search_page = home_page.search_for(name)
+    assert name == search_page.results[0].name
+    assert name in selenium.title
+
+
+@pytest.mark.native
+@pytest.mark.nondestructive
+@pytest.mark.parametrize('category, sort_attr', [
+    ['Most Users', 'users'], ['Top Rated', 'rating']])
+def test_sorting_by(
+        my_base_url, selenium, addon, minimal_addon, category, sort_attr):
+    """Test searching for an addon and sorting."""
+    page = Home(selenium, my_base_url).open()
+    name = page.featured_extensions.extensions[0].name
+    search_page = page.search_for(name)
+    search_page.sort_by(category)
+    results = [getattr(i, sort_attr) for i in search_page.results]
+    assert sorted(results, reverse=True) == results
```

Comment:
See https://addons.mozilla.org/en-US/firefox/search/?q=term for an example when sorting by users includes a result that doesn't disclose the users (Wordpot shows weekly downloads instead). Maybe this is a bug? If not, it could cause a failure when running against a live deployment when the top featured extension returns multiple results. I don't think we have to fix this now, but it's good to be aware of it.